src/api/account.ts

```diff
@@ -890,6 +890,31 @@ export class Account {
     return parseInt(balanceStr, 10);
   }
 
+  /**
+   * Retrieves the balance for an account and asset.
+   *
+   * @param args The parameters for the balance query.
+   * @param args.accountAddress The account address to query.
+   * @param args.asset The asset to query: Move struct ID (e.g., `0x1::aptos_coin::AptosCoin`) or FA metadata address.
+   * @returns The balance as a number.
+   *
+   * @example
+   * ```ts
+   * const aptos = new Aptos(new AptosConfig());
+   * // APT coin by type
+   * const apt = await aptos.getBalance({ accountAddress: "0x1", asset: "0x1::aptos_coin::AptosCoin" });
+   * // Some FA by metadata address
+   * const fa = await aptos.getBalance({ accountAddress: "0x1", asset: "0xA" });
```

Comment:
```suggestion
   * const fa = await aptos.getBalance({ accountAddress: "0x1", asset: "0xa" });
```
Nit, we tend to represent these in lower case.